lib/redux/reducers/devices.js

```diff
@@ -25,6 +25,21 @@ const devices = {
     source: {type: 'device', driverId: 'Medtronic'},
     enabled: {mac: true, win: true}
   },
+  medtronic600: {
+    instructions: 'Connect your Bayer Contour Next Link 2.4 to your computer',
```

Comment:
Might need to change wording here. I have no idea how to determine if my Next Link is "2.4" or not. (I assume that is a version number, but it isn't clear.)